recipe_scrapers/quitoque.py

```diff
@@ -0,0 +1,92 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class QuiToque(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "quitoque.fr"
+
+    @staticmethod
+    def _get_text(element):
+        if element:
+            return normalize_string(element.get_text())
+        else:
+            return None
+
+    def _get_time(self, time_name):
+        times = self.soup.select("div,.recipe-infos-short .item-info")
+        total_time = None
+        for time in times:
+            if time_name in time.get_text():
+                total_time = self._get_text(time).replace(time_name, "")
+        return get_minutes(total_time)
+
+    def _get_nutrients(self, nutrient_name):
+        nutrient_element = self._nutrient_list.find("p", string=nutrient_name).parent
+        return self._get_text(nutrient_element.find("p", class_="regular"))
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"}).get("content")
+
+    def author(self):
+        return "QuiToque"
+
+    def title(self):
+        return self._get_text(self.soup.find("h1", class_="title-2"))
+
+    def keywords(self):
+        product_tags = self.soup.find(id="product-tags").find_all(class_="badge")
+        keywords = {self._get_text(tag) for tag in product_tags}
+        return list(sorted(keywords))
+
+    def category(self):
+        category = self.soup.find(class_="primary-ghost")
+        return self._get_text(category)
+
+    def total_time(self):
+        return self._get_time("Total")
+
+    def prep_time(self):
+        return self._get_time("En cuisine")
+
+    def yields(self):
+        serving = self.soup.find(id="ingredients").find("p", class_="body-2")
+        return get_yields(serving)
+
+    def image(self):
+        img_element = self.soup.find(class_="image").find("img")
+        return img_element["src"]
+
+    def ingredients(self):
+        ingredients_list = self.soup.select("#ingredients .ingredient-list li")
+        ingredients_list.extend(self.soup.select(".kitchen-list li"))
```

Comment:
For readability, I would suggest separating these lines of code:

```suggestion
        ingredients = []
        ingredients.extend(self.soup.select("#ingredients .ingredient-list li"))
        ingredients.extend(self.soup.select("#ingredients .kitchen-list li"))
```